Library/Homebrew/extend/os/bottles.rb

```diff
@@ -0,0 +1,5 @@
+require "utils/bottles"
```

Comment:
Feels odd that this `require`s `utils/bottles` which in turn `require`s this file. This would be a cyclic dependency if `require` wasn't more sophisticated. Is this really intended? If I'm not mistaken, this has the effect that `require`ing either of those will bring in all of `Utils::Bottles`, but I feel like this file here should never be required outside of `utils/bottles.rb`. (I hope this wasn't confusing.)

If you agree, a bold way to enforce this instead of silently doing the supposedly right thing would be to fail in this line if `Utils::Bottles` hasn't been defined yet.